File: wellness/wellness.Model/Membership/Membership.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace wellness.Model.Membership
{
    public class Membership
    {
        public int Id { get; set; }

        public string ExpirationDate { get; set; } = null!;

        public string StartDate { get; set; } = null!;

        public bool Status { get; set; }

        public string UserName { get; set; } = null!;
        public float Price { get; set; }

        public string? MemberShipTypeName { get; set; }
    }
}

File: wellness/UI/mobile/lib/models/membership.dart
import 'package:json_annotation/json_annotation.dart';

part 'membership.g.dart';

@JsonSerializable()
class Membership {
  final int id;
  final String expirationDate;
  final String startDate;
  final bool status;
  final String userName;
  final String memberShipTypeName;
  final double price;
  Membership(this.id, this.expirationDate, this.startDate, this.status,
      this.userName, this.memberShipTypeName, this.price);

  factory Membership.fromJson(Map<String, dynamic> json) =>
      _$MembershipFromJson(json);

  Map<String, dynamic> toJson() => _$MembershipToJson(this);
}
